packages/datatrak-web/src/api/queries/useSubmissionHistory.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import { useQuery } from 'react-query';
+import { DatatrakWebSubmissionHistoryRequest, UserAccount, Project } from '@tupaia/types';
+import { useCurrentUser } from '../CurrentUserContext';
+import { get } from '../api';
+
+export const useSubmissionHistory = (userId?: UserAccount['id'], projectId?: Project['id']) => {
```

Comment:
I don't think this really needs to be renamed either. I think the more generic name of useRecentSurveys would still be better